lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -35,20 +38,21 @@ def check_output2(*popenargs, **kwargs):
     if 'input' in kwargs:
         if 'stdin' in kwargs:
             raise ValueError('stdin and input arguments may not both be used.')
-        inputdata = kwargs['input']
+        inputdata = to_bytes(kwargs['input'])
```

Comment:
Since we don't want encoding errors to go unnoticed here, use
``` python
inputdata = to_bytes(kwargs['input'], errors='surrogate_or_strict')
```

That will raise an error if there are encoding problems instead of silently changing the data.